Dockerfile

```diff
@@ -123,6 +123,26 @@ RUN python3 /workspace/onnxruntime/tools/ci_build/build.py --build_dir /workspac
             --update \
             --build
 
+############################################################################
```

Comment:
I thought you could just do an extra build step in the caffe2 container above?

Reply:
Yes. Moving the build to inside the caffe2 container

Reply:
Solved the build error. The libtorch build failed because it was already in the pytorch container. Tested that and it was compatible.